G1ANT.Addon.Ocr.Google/G1ANT.Addon.Ocr.Google.Tests/Commands/OcrGoogle.FindPoint/ShouldReturnPointPosition_WhenTextIsFound.G1ANT

```diff
@@ -1,11 +1,13 @@
 ﻿addon iexplorer version 2.205.17304.1000
 addon ocrgoogle version 2.205.17304.1000
 
+process.kill iexplore
 ie.open http://digitalnativestudios.com/textmeshpro/docs/rich-text/lowercase-uppercase-smallcaps.png
 window ‴✱Internet Explorer✱‴ style maximize
+delay milliseconds 200
 ocrgoogle.login ♥credential⟦ocrgoogle:jsoncredential⟧
 ocrgoogle.findpoint ‴alice‴ area ♥screen result ♥pointResult
 ie.close
 
 test ⊂♥pointResult⟦x⟧.GetType() == typeof(Int32)⊃
-test ⊂♥pointResult⟦y⟧.GetType() == typeof(Int32)⊃
\ No newline at end of file
+test ⊂♥pointResult⟦y⟧.GetType() == typeof(Int32)⊃
```

Comment:
same here (checking the type of the result, not the result)

Reply:
Check here is that the values are gt 0.

Reply:
done